### PR TITLE
Update economics

### PR DESCRIPTION
**Pull request by kmanc**
Mini rewrite to shorten things and keep tone consistent with blockchain directory

### CODE REVIEW

**Review by Suhmedoh — APPROVED** [review]
looks good, there were a couple grammar/typos that I fixed, I think I got all of them

File: education/cryptocurrency/economics.md [hunk 1]
```diff
@@ -1,33 +1,31 @@
...
 
 #### Are cryptocurrencies a limited resource?
-For most cryptocurrencies, there is a limited number of "coins" that can be available at any given time. You can't simply "create" a bitcoin and add it to your wallet, due to cryptography and transaction logging. Coins are currently created by "miners" who form the backbone of any cryptocurrency. Mining often requires much more computing power than the average person has access to, but fortunately for us, we do not need to be a miner in order to use cryptocurrency.
```
Comment (nkmlombardi) [inline education/cryptocurrency/economics.md]: The end of this paragraph is unnatural to read for me.

File: education/cryptocurrency/economics.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Let's use an example to show what makes a cryptocurrency valuable.
...
 
 #### Are cryptocurrencies a limited resource?
-For most cryptocurrencies, there is a limited number of "coins" that are available at any given time. You can't just "create" a bitcoin out of nowhere and add it to your wallet. Coins are created by "miners" who are kinda like a government's mint. Because miners can only create a limited number of coins in most cryptocurrencies, the coins are a limited resource.
+For most cryptocurrencies, there is a limited number of "coins" that are available at any given time. You can't just "create" a bitcoin out of nowhere and add it to your wallet. Coins are created by "miners" who are kind of like a government's mint. Miners can only create a limited number of coins in most cryptocurrencies, so the coins are a limited resource.
```
Comment (nkmlombardi) [inline education/cryptocurrency/economics.md]: The end of this paragraph is still unnatural for me. I'd advise against the use of the word mint.

Comment (Suhmedoh) [reply education/cryptocurrency/economics.md]: "...kind of like a government's mint(which creates the coins used for currency)."

Better? Mint is the proper name of a facility that produces coinage, afaik

Comment (nkmlombardi) [reply education/cryptocurrency/economics.md]: I propose:

> kind of like a government's mint (the creation and entrance of new coins into circulation).

Comment (kmanc) [reply education/cryptocurrency/economics.md]: I like Josh's better. Nick yours is technically correct, but sounds unnatural and overly complicated to me

Comment (nkmlombardi) [reply education/cryptocurrency/economics.md]: @kmanc What about something more in this direction:
> For most cryptocurrencies, there is a limited number of coins that are available at any given time. You can't just "create" a bitcoin out of nowhere and add it to your wallet. New coins are created by "mining", which is similar to a government printing new money. Mining creates coins at a predetermined rate however, and is not controlled by anyone. Some currencies have a maximum number of coins that can ever be created, making them a limited resource.

Comment (kmanc) [reply education/cryptocurrency/economics.md]: I like that. I think "the way a government prints new money" might flow better but either one is cool with me

**Pull request merged**